db/re/item_combo_db.txt

```diff
@@ -218,7 +218,7 @@
 2527:5467:13030,{ bonus2 bAddRace,RC_Dragon,25; }
 2527:13001,{ bonus2 bAddRace,RC_Dragon,5; }
 2527:13030,{ bonus2 bAddRace,RC_Dragon,5; }
-2584:5579,{ .@r = getequiprefinerycnt(EQI_HEAD_TOP); .@bonus = max(getskilllv("TF_DOUBLE"),5); if(.@r>=5) { skill "TF_DOUBLE",1; bonus bDoubleRate,25; } else if(.@r>=7) { skill "TF_DOUBLE",.@bonus; bonus bDoubleRate,.@bonus*5; } }
+2584:5579,{ .@r = getequiprefinerycnt(EQI_GARMENT); .@bonus = max(getskilllv("TF_DOUBLE"),5); if(.@r>=5) { skill "TF_DOUBLE",1; bonus bDoubleRate,25; } else if(.@r>=7) { skill "TF_DOUBLE",.@bonus; bonus bDoubleRate,.@bonus*5; } }
```

Comment:
In the previous review I mean to move `.@bonus` in the >= 7 condition since it's useless in the +5 condition
```
2584:5579,{ .@r = getequiprefinerycnt(EQI_GARMENT); if(.@r>=5) { skill "TF_DOUBLE",1; bonus bDoubleRate,25; } else if(.@r>=7) { .@bonus = max(getskilllv("TF_DOUBLE"),5);  skill "TF_DOUBLE",.@bonus; bonus bDoubleRate,.@bonus*5; } }
```

Reply:
Oh now I understood.